Documentation/configuring-hive-metastore.md

```diff
@@ -35,20 +35,20 @@ Use [metastore-storage.yaml][metastore-storage-config] as a template and adjust
 By default to make installation easier Metering configures Hive to use an embedded Java database called [Derby](https://db.apache.org/derby/#What+is+Apache+Derby%3F), however this is unsuited for larger environments or metering installations with a lot of reports and metrics being collected.
 Currently two alternative options are available, MySQL and PostgreSQL, both of which have been tested with the metering-operator.
 
-There are 4 configuration options you can use to control the database used by Hive metastore: `url` , `driver` , `username` , and `password`.
+There are 4 configuration options you can use to control the database used by Hive metastore: `url` , `driver` , and `secretName` .
```

Comment:
This may not be in the scope of this PR, but we may want to add some more description to the various configurable fields, besides just listing them off in succession and expect users can fill in the gaps themselves.